test/wasm/i32_unreachable.js

```diff
@@ -0,0 +1,18 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+const blob = WScript.LoadBinaryFile('i32_unreachable.wasm');
+const moduleBytesView = new Uint8Array(blob);
+var a = Wasm.instantiateModule(moduleBytesView, {}).exports;
+
+try {
+print(a.test_unreachable());
+} catch (e) {
+    print("Caught the following exp:");
```

Comment:
Instead of having a baseline here.
We could simply do 
```js
try{
  a.test_unreachable();
  print("FAILED");
catch(e) {
  print(e.message.includes("Unreachable") ? "PASSED" : "FAILED");
}
```